modules/search/class.jetpack-search.php

```diff
@@ -282,6 +283,28 @@ public function load_assets() {
 		}
 	}
 
+	/**
+	 * Is search supported on the current plan
+	 *
+	 * @since 6.0
+	 * Loads scripts for Tracks analytics library
+	 */
+	public function is_search_supported() {
+		return Jetpack::active_plan_supports( 'search' );
+	}
+
+	/**
+	 * Does this site have a VIP index
+	 * Get the version number to use when loading the file. Allows us to bypass cache when developing.
+	 *
+	 * @since 6.0
+	 * @param string $file Path of the file we are looking for.
+	 * @return string $script_version Version number.
+	 */
+	public function has_vip_index() {
```

Comment:
This function already exists (as `site_has_vip_index()`) in class.jetpack-search-helpers.php, would be best to settle on one or the other. Also I think the phpdoc comments are wrong.

Reply:
Similar to is_search_supported() we override this in class.wpcom-search.php on wp.com and I would rather us only have one class that we are overriding.

It looks like site_has_vip_index() is not getting called anywhere in Jetpack. Is that true? Or do VIPs use this elsewhere?

And ya phpdoc comments are very wrong. Will fix...

Reply:
I don't see any instances of VIPs using `site_has_vip_index()` directly.

Reply:
I've left this as is for now since it doesn't really affect anything and it is in sync.